src/tv2_afvd_showstyle/migrations/index.ts

```diff
@@ -4,6 +4,8 @@ import {
 	changeGFXTemplate,
 	GetDefaultAdLibTriggers,
 	GetDSKSourceLayerNames,
+	mapGFXTemplateToDesignTemplateAndDeleteOriginals,
+	// removeDesignChangesFromGFXTemplate,
```

Comment:
A wild comment appeared, would be nice if it was deleted. 
This seems to be your old migration that you have refactored. If that is true, then this is a marvelous example of how fast comments becomes outdated. This one didn't even make it to develop ;)

Reply:
Yeah i wonder who put that there *coughs nervously*....